app-office/planner/planner-9999.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="Project manager for Gnome"
+HOMEPAGE="https://wiki.gnome.org/Apps/Planner"
```

Comment:
Could you use this live template (if ...) in the release version too?

Reply:
Ah, I see it'll still be somewhat out of sync b/c meso nvs autotools, but we can copy this bit at least.

Reply:
I thought once upstream releases a new version we drop the old ebuild and use the 9999 ebuild for the release. 
Right now I do not really want to use implement the 'if PV=9999' thing in the release ebuild, as it would have to switch between meson and gnome2 builds everywhere.

Reply:
Sorry, what I meant was, just do it for SRC_URI at the top.